third_party/ml-metadata/README.md

```diff
@@ -24,18 +24,29 @@ Run update_dependencies.sh in the following way:
 
 Make sure the generated files are as expected. Update clients as described below:
 
-## Build golang gRPC client from proto
+### Build golang gRPC client from proto
 
-### Prerequisites
+#### Prerequisites
 
 Make sure you have installed tools and packages in [grpc golang prerequisites](https://grpc.io/docs/languages/go/quickstart/#prerequisites).
 
-### Command
+#### Command
 
 ```bash
 make
 ```
 
-## Build JS client from proto
+#### Adopt new golang client in backend
+
+After submitting the changes from above command, find all the modules with `github.com/kubeflow/pipelines/third_party/ml-metadata` in all `go.mod` files. Update them by the following steps:
+
+1. Run `go list github.com/kubeflow/pipelines/third_party/ml-metadata`
+1. Write down the new version `vX.X.X-date-hash` on master branch. (Example: `v0.0.0-20211222053840-ef8e092d2896`)
+1. Download new module in corresponding folder: `go get github.com/kubeflow/pipelines/third_party/ml-metadata@{new-version}`, for example: `go get github.com/kubeflow/pipelines/third_party/ml-metadata@v0.0.0-20211222053840-ef8e092d2896`.
```

Comment:
have you considered just running `go get github.com/kubeflow/pipelines/third_party/ml-metadata@latest` instead of these?

Reply:
That is a good point, but the latest is subject to change, therefore whenever you build image, it will be different because `@latest` is pointing to different codebase. Does it make sense?

Reply:
Note, when you get latest, it pins to current latest version in go.mod and go.sum. It won't change unless someone go gets again.

Reply:
I see, sounds good and updated to using `go get ...@latest`.